### PR TITLE
Fixed small error in Akka Typed Getting Started Tutorial 5

### PR DESCRIPTION
**Pull request by twbraam**
## Purpose

In the Akka Typed Getting Started Guide there is a small discrepancy between the code and text in Part 5 (there is no mention of updating the RespondTemperature message). This PR closes the gap.

## References

No references

## Changes

* Added text to the tutorial_5 text and the relevant code snippets.

## Background Context

Followed the Getting Started Guide and implementing the changes in the last part will cause the project to not compile.

### CODE REVIEW

**Comment by johanandren** [comment]
OK TO TEST

File: akka-docs/src/test/java/jdocs/typed/tutorial_5/Device.java [hunk 1]
```diff
@@ -53,10 +53,12 @@ public ReadTemperature(long requestId, ActorRef<RespondTemperature> replyTo) {
     final String deviceId;
     final Optional<Double> value;
 
+    // #respond-declare
     public RespondTemperature(long requestId, String deviceId, Optional<Double> value) {
       this.requestId = requestId;
       this.deviceId = deviceId;
       this.value = value;
+      // #respond-declare
```
Comment (johanandren) [inline akka-docs/src/test/java/jdocs/typed/tutorial_5/Device.java]: I think this looked a bit weird in the page, with only pulling in the constructor, let's make it the entire class for Java.

**Review by johanandren — COMMENTED** [review]
Thanks, looks like a nice little improvement. One minor thing noted.

File: akka-docs/src/test/java/jdocs/typed/tutorial_5/Device.java (hunk 1 above)
Comment (twbraam) [reply akka-docs/src/test/java/jdocs/typed/tutorial_5/Device.java]: Made the change, how's this?

Comment (johanandren) [reply akka-docs/src/test/java/jdocs/typed/tutorial_5/Device.java]: Sorry, I should have been more clear in my feedback. 

I meant that `// #respond-declare`  should be the entire class `RespondTemperature` in Java, just like it is in Scala. I think it is fine like it was before except that.

Comment (twbraam) [reply akka-docs/src/test/java/jdocs/typed/tutorial_5/Device.java]: No worries, my Java skills are not all that so I failed to see the class borders.

**Review by johanandren — APPROVED** [review]
LGTM, thanks!

**Review by patriknw — APPROVED** [review]
LGTM, thanks

**Pull request merged**